application/src/test/java/org/thingsboard/server/controller/AbstractWebTest.java

```diff
@@ -25,9 +25,11 @@
 import io.jsonwebtoken.Jwt;
 import io.jsonwebtoken.Jwts;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpStatus;
 import org.awaitility.Awaitility;
 import org.hamcrest.Matcher;
 import org.hibernate.exception.ConstraintViolationException;
+import org.jetbrains.annotations.NotNull;
```

Comment:
It seems unused

Reply:
@smatvienko-tb , please create an automatic check to fail the build if the code has "org.jetbrains.annotations".